helmfile.d/0120.cert-manager.yaml

```diff
@@ -0,0 +1,39 @@
+helmDefaults:
+  args:
+    - "--wait"
+    - "--recreate-pods"
+    - "--timeout=600" 
+    - "--force"
+    - "--reset-values"
+
+repositories:
+# Stable repo of official helm charts
+- name: "stable"
+  url: "https://kubernetes-charts.storage.googleapis.com"
+
+
+releases:
+
+################################################################################
+## CERT-MANAGER - Automatic Let's Encrypt for Ingress  #########################
+################################################################################
+
+- name: "cert-manager"
+  namespace: "kube-system"
+  labels:
+    chart: "cert-manager"
+    component: "ingress"
+    namespace: "kube-system"
+    vendor: "jetstack"
+    default: "true"
```

Comment:
default: "false"
we have two cert managers now (kube-lego id another one).
we need to make both of them not default

Reply:
Ok, what I should have said, is we need one to be true. Let's make this one false and leave `kube-lego` the default until we document how to use `cert-manager.